src/test/resources/cucumber.features/Ex1DifferentElementsPage.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Different Elements Page logging feature
+
+  Scenario: Different Elements Page. Selecting elements.
+
+
+    Given I open JDI GitHub site
+    And I login as user "Roman Iovlev"
+    And I click on 'Service' button in Header
+    And I click on 'Different Elements' button in Service dropdown
+
+    When I select 'Water' checkbox on Different Elements Page
+    And I select 'Wind' checkbox on Different Elements Page
+    And I select 'Selen' radio button on Different Elements Page
+    And I select 'Yellow' in dropdown on Different Elements Page
+
+    Then The top log row should be containing 'Water: condition changed to true' text in log section
+    And The top log row should be containing 'Wind: condition changed to true' text in log section
+    And The top log row should be containing 'metal: value changed to Selen' text in log section
+    And The top log row should be containing 'Colors: value changed to Yellow' text in log section
```

Comment:
it's better to create one scenario and pass wind/water/colors as a parameters

Reply:
did not get this one, because we need to check in one exercise==scenario
•	for each checkbox there is an individual log row and value is corresponded to the status of checkbox
•	for radio button there is a log row and value is corresponded to the status of radio button
•	for dropdown there is a log row and value is corresponded to the selected value.